ChangeLog.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [v0.10.2] - 2019-07-24
+### Fixed
+- `All APIs`: prevent `meta` field of the `$requestPacket` from being overwritten when SDK Telemetry is enabled.
```

Comment:
I don't think we need to mention "All APIs" here. The `DataAPI` mention in the previous log entry is just referring to a specific set of methods in the DataApi class of the SDK.